docs/streams/developer-guide.html

```diff
@@ -488,6 +488,24 @@
         So we can avoid the situation where one instance is assigned all tasks, begins restoring/processing, only to shortly after be rebalanced, and then have to start again with half of the tasks and so on.
     </p>
 
+    <h4><a id="streams_topic_config" href="#streams_topic_config">Internal Topic Configuration</a></h4>
+    <p>
+        Kafka Streams automatically creates internal repartitioning and changelog topics.
+        You can override the configs used when creating these topics by adding any configs from <code>TopicConfig</code> to your <code>StreamsConfig</code> with the prefix <code>topic.</code>:
```

Comment:
Instead of advocating to use the hard-coded string value, we can just use `StreamsConfig .TOPIC_PREFIX` here? And ditto at line 503. Also we'd probably mention that for changlelog topics, users can also override these global configs on the per-store basis when specifying their topologoy.

Otherwise LGTM.